src/frontends/arcs/xform_to_datalog/datalog_facts_test.cc

```diff
@@ -64,10 +64,8 @@ INSTANTIATE_TEST_SUITE_P(
         std::make_tuple(ManifestDatalogFacts(),
                         *(AuthorizationLogicDatalogFacts::create(
                             utils::test::GetTestDataDir(path),
-                            "empty_auth_logic")),
-                        R"(// GENERATED FILE, DO NOT EDIT!
```

Comment:
Not a big deal, but is there any specific reason to get rid of this comment?

Reply:
Yes! I had mistaken some test files for source files. I think I've fixed them now, thanks for the reminder.